README.md

```diff
@@ -46,6 +46,8 @@ use {"akinsho/org-bullets.nvim", config = function()
   require("org-bullets").setup {
     concealcursor = false, -- If false then when the cursor is on a line underlying characters are visible
     symbols = {
+      -- list_bullet symbol
+      list_bullet = "•",
```

Comment:
Is the `_bullet` necessary since this is already nested in a `symbols` table? I don't think there is any other type of list icon so `symbols = { list = ... }` seems fine to me

Reply:
list_bullet changed to list!